docs/source/science_examples.rst

```diff
@@ -1,6 +1,10 @@
 Science Examples
 =======================================
 
+Here you will find a variety of real-world examples that may help you create a custom Jupyter notebook for your research. These notebooks have data-set specific analysis and visualization examples, and will also provide examples of analysis, data access, tabulation and map-visualization.
```

Comment:
data-set or dataset?

Reply:
dataset is more consistent with the rest of the docs
changed